index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/0278cc9d4c.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;700&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="style.css">
+    <title>John Doe</title>
+</head>
+<body>
+    <nav class="navbar">
+        <div class="navbar__logo">
+            John Doe
+        </div>
+        <div class="navbar__menu">
+            <i class="fa-solid fa-bars"></i>
+        </div>
+    </nav>
```

Comment:
- [ ] Kindly wrap your `nav` tag with `header` section, because this is your header.
- [ ] Kindly use `a` tag instead of `div` tag with your logo, and make it link to your home page.